From c1b4101464c5956ae2cb9f31e21678e66a21aa5f Mon Sep 17 00:00:00 2001
From: Nizan Zorea <nzorea@marvell.com>
Date: Wed, 11 Nov 2015 14:49:07 +0200
Subject: [PATCH 0495/1240] fix: thermal: update timeout for checking the valid
 bit

	- Update the delay in loop that checking the valid bit.
	- Before: the loop ended before the valid bit was enabled.
	- JIRA SYSTEMSW-2025

Change-Id: I3c433695ae2a47815e8cb550b918b616f7fe28c9
Signed-off-by: Nizan Zorea <nzorea@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/24738
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 drivers/thermal/mvebu_thermal_28nm_v2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/thermal/mvebu_thermal_28nm_v2.c b/drivers/thermal/mvebu_thermal_28nm_v2.c
index 6573929..90b56ca 100644
--- a/drivers/thermal/mvebu_thermal_28nm_v2.c
+++ b/drivers/thermal/mvebu_thermal_28nm_v2.c
@@ -121,7 +121,7 @@ u32 mvebu_thermal_sensor_probe(void)
 
 	reg = readl(tsen->regs_base + THERMAL_SEN_CTRL_STATS);
 	while ((reg & THERMAL_SEN_CTRL_STATS_VALID_MASK) == 0 && timeout < THERMAL_TIMEOUT) {
-		udelay(1);
+		udelay(10);
 		reg = readl(tsen->regs_base + THERMAL_SEN_CTRL_STATS);
 		timeout++;
 	}
-- 
1.9.1

